console/console-init/ui/src/components/DropdownWithBulkSelect/DropdownWithBulkSelect.tsx

```diff
@@ -16,10 +16,10 @@ export interface IDropdownWithBulkSelectProps {
   checkBoxId: string;
   ariaLabel: string;
   isChecked: boolean;
-  isOpen: boolean;
-  handleOnToggle: (val: boolean) => void;
-  items: React.ReactNode[];
-  handleOnSelect: () => void;
+  isOpen?: boolean;
```

Comment:
I think we don't need to add prefix handle with event name, onToggle, onSelect,onChnage names are good enough.